packages/connect-conformance/package.json

```diff
@@ -9,9 +9,12 @@
   "scripts": {
     "generate": "buf generate buf.build/connectrpc/conformance:v1.0.0-rc2",
     "clean": "rm -rf ./dist/cjs/*",
-    "build": "tsc --project tsconfig.json --module commonjs --outDir ./dist/cjs",
+    "build": "npm run build:cjs && npm run build:esm",
+    "build:cjs": "tsc --project tsconfig.json --module commonjs --outDir ./dist/cjs",
+    "build:esm": "tsc --project tsconfig.json --module ES2015 --outDir ./dist/esm",
     "test:node:server": "./bin/connectconformance --mode server --conf conformance-node.yaml -v ./bin/conformancenodeserver",
-    "test:node:client": "./bin/connectconformance --mode client --conf conformance-node.yaml -v ./bin/conformancenodeclient"
+    "test:node:client": "./bin/connectconformance --mode client --conf conformance-node.yaml -v ./bin/conformancenodeclient",
+    "test:cloudflare:server": "npx wrangler deploy -c wrangler-server.toml && ./bin/connectconformance --mode server --conf conformance-cloudflare-server.yaml -v ./bin/conformancecloudflareserver"
```

Comment:
Following [server_compat.proto](https://github.com/connectrpc/conformance/blob/main/proto/connectrpc/conformance/v1/server_compat.proto), I would expect `conformancecloudflareserver` to deploy.

This would allow us to honor `message_receive_limit`.

I suppose we don't because deploy time can cause the runner to time out? Are there other reasons?

Reply:
There could be multiple servers running in parallel

Reply:
With the added comment in packages/connect-conformance/src/cloudflare/server.ts, this works for me 👍